### ISSUE TITLE
After deleting the default committee, I cannot create a new committee

### ISSUE THREAD

**Issue by GabrielInTheWorld** [OpenSlides/openslides-backend#1085]
I sent this request:
```js
[
    {
        "action": "committee.create",
        "data": [
            {
                "name": "Pokémonwelt",
                "organization_id": 1,
                "description": "",
                "organization_tag_ids": [],
                "manager_ids": [
                    1
                ],
                "forward_to_committee_ids": [],
                "receive_forwardings_from_committee_ids": []
            }
        ]
    }
]
```

and got this error:
```js
{
    "success": false,
    "message": "Datastore service sends HTTP 400. Model 'committee/1' does not exist."
}
```

Logs from the backend:
```
docker-backend-1           | DEBUG:openslides_backend.http.views.base_view:User id is 1.
docker-backend-1           | DEBUG:openslides_backend.shared.handlers.base_handler:Perform action committee.create.
docker-backend-1           | DEBUG:openslides_backend.services.datastore.adapter:Start GET request to datastore with the following data: GetRequest(fqid='user/1', mapped_fields={'organization_management_level', 'meta_position'}, position=None, get_deleted_models=<DeletedModelsBehaviour.NO_DELETED: 1>)
docker-backend-1           | DEBUG:openslides_backend.services.datastore.adapter:Start RESERVE_IDS request to datastore with the following data: Collection: committee, Amount: 1
docker-backend-1           | DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): datastore-writer:9011
docker-datastore-writer-1  | 2021-12-08T08:29:58.398: [/app/datastore/writer/core/writer_service.py] [INFO] 1 ids reserved
docker-datastore-writer-1  | 172.18.0.13 - - [08/Dec/2021 08:29:58] "POST /internal/datastore/writer/reserve_ids HTTP/1.1" 200 -
docker-backend-1           | DEBUG:urllib3.connectionpool:http://datastore-writer:9011 "POST /internal/datastore/writer/reserve_ids HTTP/1.1" 200 11
docker-backend-1           | DEBUG:openslides_backend.services.datastore.adapter:Get response with status code 200: {'ids': [2]}
docker-backend-1           | DEBUG:openslides_backend.services.datastore.adapter:Start GET request to datastore with the following data: GetRequest(fqid='user/1', mapped_fields={'is_active', 'meta_position'}, position=None, get_deleted_models=<DeletedModelsBehaviour.NO_DELETED: 1>)
docker-backend-1           | DEBUG:openslides_backend.services.datastore.adapter:Start GET request to datastore with the following data: GetRequest(fqid='user/1', mapped_fields={'meta_position', 'committee_$_management_level'}, position=None, get_deleted_models=<DeletedModelsBehaviour.NO_DELETED: 1>)
docker-backend-1           | DEBUG:openslides_backend.services.datastore.adapter:Start GET request to datastore with the following data: GetRequest(fqid='user/1', mapped_fields={'meta_position', 'committee_ids', 'group_$_ids', 'committee_$_management_level'}, position=None, get_deleted_models=<DeletedModelsBehaviour.NO_DELETED: 1>)
docker-backend-1           | DEBUG:openslides_backend.services.datastore.adapter:Start GET_MANY request to datastore with the following data: GetManyRequest(requests=[GetManyRequestPart(collection='meeting', ids=[], mapped_fields=['meta_position', 'committee_id'])], mapped_fields=[], position=None, get_deleted_models=<DeletedModelsBehaviour.NO_DELETED: 1>)
docker-backend-1           | DEBUG:openslides_backend.services.datastore.adapter:Start GET request to datastore with the following data: GetRequest(fqid='committee/2', mapped_fields={'meta_position', 'organization_id'}, position=None, get_deleted_models=<DeletedModelsBehaviour.NO_DELETED: 1>)
docker-backend-1           | DEBUG:openslides_backend.services.datastore.adapter:Start GET request to datastore with the following data: GetRequest(fqid='organization/1', mapped_fields={'meta_position', 'committee_ids'}, position=None, get_deleted_models=<DeletedModelsBehaviour.NO_DELETED: 1>)
docker-backend-1           | DEBUG:openslides_backend.services.datastore.adapter:Start GET request to datastore with the following data: GetRequest(fqid='committee/2', mapped_fields={'meta_position', 'organization_tag_ids'}, position=None, get_deleted_models=<DeletedModelsBehaviour.NO_DELETED: 1>)
docker-backend-1           | DEBUG:openslides_backend.services.datastore.adapter:Start GET request to datastore with the following data: GetRequest(fqid='committee/2', mapped_fields={'meta_position', 'forward_to_committee_ids'}, position=None, get_deleted_models=<DeletedModelsBehaviour.NO_DELETED: 1>)
docker-backend-1           | DEBUG:openslides_backend.services.datastore.adapter:Start GET request to datastore with the following data: GetRequest(fqid='committee/2', mapped_fields={'receive_forwardings_from_committee_ids', 'meta_position'}, position=None, get_deleted_models=<DeletedModelsBehaviour.NO_DELETED: 1>)
docker-backend-1           | DEBUG:openslides_backend.shared.handlers.base_handler:Write request is ready.
docker-backend-1           | DEBUG:openslides_backend.services.datastore.adapter:Start WRITE request to datastore with the following data: Write request: [WriteRequest(events=[{'type': <EventType.Create: 'create'>, 'fqid': FullQualifiedId('committee/2'), 'fields': {'name': 'Pokémonwelt', 'organization_id': 1, 'description': '', 'organization_tag_ids': [], 'forward_to_committee_ids': [], 'receive_forwardings_from_committee_ids': [], 'id': 2}}, {'type': <EventType.Update: 'update'>, 'fqid': FullQualifiedId('user/1'), 'list_fields': {'add': {'committee_ids': [1, 2]}}}, {'type': <EventType.Update: 'update'>, 'fqid': FullQualifiedId('committee/1'), 'list_fields': {'add': {'user_ids': [1]}}}, {'type': <EventType.Update: 'update'>, 'fqid': FullQualifiedId('committee/2'), 'list_fields': {'add': {'user_ids': [1]}}}, {'type': <EventType.Update: 'update'>, 'fqid': FullQualifiedId('user/1'), 'fields': {'committee_$_management_level': ['1', '2'], 'committee_$2_management_level': <CommitteeManagementLevel.CAN_MANAGE: 'can_manage'>}}, {'type': <EventType.Update: 'update'>, 'fqid': FullQualifiedId('organization/1'), 'fields': {'committee_ids': [2]}}], information={FullQualifiedId('user/1'): ['Object updated', 'Object updated'], FullQualifiedId('committee/1'): ['Object updated'], FullQualifiedId('committee/2'): ['Object updated', 'Object created'], FullQualifiedId('organization/1'): ['Object attached to organization']}, user_id=1, locked_fields={'user/1/organization_management_level': 3, 'user/1/is_active': 3, 'user/1/committee_$_management_level': 3, 'user/1/committee_ids': 3, 'user/1/group_$_ids': 3, 'organization/1/committee_ids': 3}, migration_index=None)]
docker-backend-1           | DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): datastore-writer:9011
docker-datastore-writer-1  | 2021-12-08T08:29:58.451: [/app/datastore/writer/postgresql_backend/sql_database_backend_service.py] [INFO] Created position 4
docker-datastore-writer-1  | 2021-12-08T08:29:58.460: [/app/datastore/shared/flask_frontend/errors.py] [DEBUG] HTTP error 400: {'type': 3, 'fqid': 'committee/1', 'type_verbose': 'MODEL_DOES_NOT_EXIST'}
docker-datastore-writer-1  | 172.18.0.13 - - [08/Dec/2021 08:29:58] "POST /internal/datastore/writer/write HTTP/1.1" 400 -
docker-backend-1           | DEBUG:urllib3.connectionpool:http://datastore-writer:9011 "POST /internal/datastore/writer/write HTTP/1.1" 400 108
docker-backend-1           | DEBUG:openslides_backend.services.datastore.adapter:Get response with status code 400: {'error': {'fqid': 'committee/1', 'type': 3, 'type_verbose': 'MODEL_DOES_NOT_EXIST'}}
```

<details>
<summary>Database dump</summary>

<p>

```js
{
  "resource":{
    "1":{
      "id":1,
      "token":"web_header",
      "filesize":1462,
      "mimetype":"image/png",
      "meta_deleted":false,
      "meta_position":1,
      "organization_id":1
    }
  },
  "theme":{
    "1":{
      "id":1,
      "name":"OpenSlides Blue",
      "warn_500":"#f06400",
      "accent_500":"#2196f3",
      "primary_500":"#317796",
      "meta_deleted":false,
      "meta_position":1,
      "organization_id":1,
      "theme_for_organization_id":1
    },
    "2":{
      "id":2,
      "name":"OpenSlides Red",
      "warn_500":"#11c2a2",
      "accent_500":"#03a9f4",
      "primary_500":"#c31c23",
      "meta_deleted":false,
      "meta_position":1,
      "organization_id":1
    },
    "3":{
      "id":3,
      "name":"OpenSlides Green",
      "warn_500":"#e359ce",
      "accent_500":"#55c3b6",
      "primary_500":"#46962c",
      "meta_deleted":false,
      "meta_position":1,
      "organization_id":1
    }
  },
  "user":{
    "3":{
      "id":3,
      "email":"",
      "title":"",
      "gender":"diverse",
      "number_$":[
        
      ],
      "password":"316af7b2ddc20ead599c38541fbe87e9a9e4e960d4017d6e59de188b41b2758fIxDxvpkn6dDLRxT9DxJhZ/f04AL2oK2beICRFobSw53CI93U+dfN+w+NaL7BvrcR4JWuMj9NkH4dVjnnI0YTkg==",
      "username":"b",
      "comment_$":[
        
      ],
      "is_active":true,
      "last_name":"",
      "about_me_$":[
        
      ],
      "first_name":"b",
      "vote_$_ids":[
        
      ],
      "group_$_ids":[
        
      ],
      "meeting_ids":[
        
      ],
      "group_$1_ids":[
        
      ],
      "is_demo_user":false,
      "meta_deleted":false,
      "option_$_ids":[
        
      ],
      "committee_ids":[
        
      ],
      "meta_position":2,
      "option_$1_ids":[
        
      ],
      "speaker_$_ids":[
        
      ],
      "vote_weight_$":[
        
      ],
      "default_number":"",
      "speaker_$1_ids":[
        
      ],
      "default_password":"jKwSLGCk",
      "poll_voted_$_ids":[
        
      ],
      "projection_$_ids":[
        
      ],
      "structure_level_$":[
        
      ],
      "chat_message_$_ids":[
        
      ],
      "is_physical_person":true,
      "default_vote_weight":"1.000000",
      "personal_note_$_ids":[
        
      ],
      "submitted_motion_$_ids":[
        
      ],
      "supported_motion_$_ids":[
        
      ],
      "vote_delegated_$_to_id":[
        
      ],
      "can_change_own_password":true,
      "default_structure_level":"",
      "supported_motion_$1_ids":[
        
      ],
      "is_present_in_meeting_ids":[
        
      ],
      "vote_delegated_vote_$_ids":[
        
      ],
      "assignment_candidate_$_ids":[
        
      ],
      "assignment_candidate_$1_ids":[
        
      ],
      "vote_delegations_$_from_ids":[
        
      ],
      "committee_$_management_level":[
        
      ]
    },
    "2":{
      "id":2,
      "email":"",
      "title":"",
      "gender":"female",
      "number_$":[
        
      ],
      "password":"316af7b2ddc20ead599c38541fbe87e9a9e4e960d4017d6e59de188b41b2758fDB3tv5HcCtPRREt7bPGqerTf1AbmoKXt/fVFkLY4znDRh2Yy0m3ZjXD0nHI8oa6KrGlHH/cvysfvf8i2fWIzmw==",
      "username":"a",
      "comment_$":[
        
      ],
      "is_active":true,
      "last_name":"",
      "about_me_$":[
        
      ],
      "first_name":"a",
      "vote_$_ids":[
        
      ],
      "group_$_ids":[
        
      ],
      "meeting_ids":[
        
      ],
      "group_$1_ids":[
        
      ],
      "is_demo_user":false,
      "meta_deleted":false,
      "option_$_ids":[
        
      ],
      "committee_ids":[
        
      ],
      "meta_position":2,
      "option_$1_ids":[
        
      ],
      "speaker_$_ids":[
        
      ],
      "vote_weight_$":[
        
      ],
      "default_number":"",
      "speaker_$1_ids":[
        
      ],
      "default_password":"a",
      "poll_voted_$_ids":[
        
      ],
      "projection_$_ids":[
        
      ],
      "structure_level_$":[
        
      ],
      "chat_message_$_ids":[
        
      ],
      "is_physical_person":true,
      "default_vote_weight":"1.000000",
      "personal_note_$_ids":[
        
      ],
      "submitted_motion_$_ids":[
        
      ],
      "supported_motion_$_ids":[
        
      ],
      "vote_delegated_$_to_id":[
        
      ],
      "can_change_own_password":true,
      "default_structure_level":"",
      "is_present_in_meeting_ids":[
        
      ],
      "vote_delegated_vote_$_ids":[
        
      ],
      "assignment_candidate_$_ids":[
        
      ],
      "assignment_candidate_$1_ids":[
        
      ],
      "vote_delegations_$_from_ids":[
        
      ],
      "committee_$_management_level":[
        
      ]
    },
    "1":{
      "id":1,
      "email":"",
      "title":"",
      "gender":"male",
      "number_$":[
        
      ],
      "password":"316af7b2ddc20ead599c38541fbe87e9a9e4e960d4017d6e59de188b41b2758flD5BVZAZ8jLy4nYW9iomHcnkXWkfk3PgBjeiTSxjGG7+fBjMBxsaS1vIiAMxYh+K38l0gDW4wcP+i8tgoc4UBg==",
      "username":"admin",
      "comment_$":[
        
      ],
      "is_active":true,
      "last_name":"Administrator",
      "about_me_$":[
        
      ],
      "first_name":"",
      "vote_$_ids":[
        
      ],
      "group_$_ids":[
        
      ],
      "meeting_ids":[
        
      ],
      "vote_$1_ids":[
        
      ],
      "group_$1_ids":[
        
      ],
      "is_demo_user":false,
      "meta_deleted":false,
      "option_$_ids":[
        
      ],
      "committee_ids":[
        
      ],
      "meta_position":3,
      "option_$1_ids":[
        
      ],
      "speaker_$_ids":[
        
      ],
      "vote_weight_$":[
        
      ],
      "default_number":"",
      "speaker_$1_ids":[
        
      ],
      "default_password":"admin",
      "poll_voted_$_ids":[
        
      ],
      "projection_$_ids":[
        
      ],
      "poll_voted_$1_ids":[
        
      ],
      "structure_level_$":[
        
      ],
      "chat_message_$_ids":[
        
      ],
      "is_physical_person":true,
      "default_vote_weight":"1.000000",
      "personal_note_$_ids":[
        
      ],
      "personal_note_$1_ids":[
        
      ],
      "submitted_motion_$_ids":[
        
      ],
      "supported_motion_$_ids":[
        
      ],
      "vote_delegated_$_to_id":[
        
      ],
      "can_change_own_password":true,
      "default_structure_level":"",
      "submitted_motion_$1_ids":[
        
      ],
      "is_present_in_meeting_ids":[
        
      ],
      "vote_delegated_vote_$_ids":[
        
      ],
      "assignment_candidate_$_ids":[
        
      ],
      "vote_delegated_vote_$1_ids":[
        
      ],
      "assignment_candidate_$1_ids":[
        
      ],
      "vote_delegations_$_from_ids":[
        
      ],
      "committee_$_management_level":[
        "1"
      ],
      "committee_$1_management_level":"can_manage",
      "organization_management_level":"superadmin"
    }
  },
  "organization":{
    "1":{
      "id":1,
      "name":"Test Organization",
      "theme_id":1,
      "theme_ids":[
        1,
        2,
        3
      ],
      "login_text":"Guten Morgen!",
      "description":"",
      "enable_chat":true,
      "legal_notice":"<a href=\"http://www.openslides.org\">OpenSlides</a> is a free web based presentation and assembly system for visualizing and controlling agenda, motions and elections of an assembly.",
      "meta_deleted":false,
      "resource_ids":[
        1
      ],
      "committee_ids":[
        
      ],
      "meta_position":3,
      "limit_of_users":0,
      "privacy_policy":"",
      "limit_of_meetings":0,
      "active_meeting_ids":[
        
      ],
      "organization_tag_ids":[
        1
      ],
      "enable_electronic_voting":true,
      "reset_password_verbose_errors":true
    }
  },
  "organization_tag":{
    "1":{
      "id":1,
      "name":"Orga Tag 1",
      "color":"#317796",
      "tagged_ids":[
        
      ],
      "meta_deleted":false,
      "meta_position":3,
      "organization_id":1
    }
  }
}
```

</p>
</details>

**Comment by jsangmeister** [comment]
I _think_ this is a duplicate of https://github.com/OpenSlides/openslides-backend/issues/1071, since the only mention of `committee/1` in your dump is in the CML. So hopefully this is also fixed with #1072

**Comment by jsangmeister** [comment]
@r-peschke please check this when you are done
